database/__init__.py

```diff
@@ -206,6 +230,25 @@ def edit(token):
     pass
 
 
+@app.route('/view_data/<token>')
+def view_data(token):
+    try:
+        email = ts.loads(token, salt='request_data-key')
+
+    except SignatureExpired:
+        abort(404)
+
+    p = Person.query.filter_by(email=email).all()
```

Comment:
`.first()` reicht hier. Die E-Mail ist unique in der Datenbank, darauf kann man sich verlassen.